.github/actions/update-github-issue-and-pull-request/Update-GitHubIssueAndPullRequest.ps1

```diff
@@ -41,8 +41,14 @@ elseif ($Body -NotLike "*``[$issueKey``]``($jiraBrowseUrl$issuekey``)*")
 }
 
 $issueQuery = "$issueKey in:title"
-$output = gh issue list --search $issueQuery --repo $GitHubRepository
-$issueItem = ($output | Select-Object -First 1)
+$issueItem = $null
+$issuesEnabled = (gh repo view $GitHubRepository --json hasIssuesEnabled | ConvertFrom-Json).hasIssuesEnabled
+
+if ($issuesEnabled)
```

Comment:
```suggestion
if ((gh repo view $GitHubRepository --json hasIssuesEnabled | ConvertFrom-Json).hasIssuesEnabled)
```

Reply:
Fair. It's more compact. I just really dislike the complex condition statements directly in the if as it often makes a hard time to read.